### ISSUE TITLE
Public forms with $2sxc.cms.run outputs error

### ISSUE THREAD

**Issue by eXistenZe** [2sic/2sxc#2978]
**I'm submitting a ...**   
[x] bug report 

**...about** 
[x] Content Types or data management
[x] APIs like REST

**Current behavior**
Using this for an old app:

...
`onclick="pAddAdmin(this)"`
...

```
function pAddAdmin(tag) {
    $2sxc(tag).cms.run({ action: "new", params: { contentType: "administration"} })
}
```

OR

```
function pAddAdmin(tag) {
    $2sxc.cms.run(tag, "new", { contentType: "administration"})
}
```

AND

`@Edit.Enable(api: true, forms: true, context: true, autoToolbar: false)`

**Expected behavior**
This worked for the last couple years, but some update broke this funcionallity.

**Instructions to Reproduce the Problem**
Calling the funcion will result in:
```
core.mjs:6485 ERROR Error: Context is missing some of the required parameters
    at m.initRoot (context.ts:81:13)
    at new _ (app.component.ts:37:18)
    at _.ɵfac [as factory] (app.component.ts:65:23)
    at ss (core.mjs:3565:44)
    at Y_ (core.mjs:10179:23)
    at dh (core.mjs:12280:23)
    at Oc.create (core.mjs:21633:25)
    at e.bootstrap (core.mjs:26489:42)
    at core.mjs:26172:64
    at Array.forEach (<anonymous>)
Hm @ core.mjs:6485
handleError @ core.mjs:6532
(anonymous) @ core.mjs:26246
invoke @ zone.js:372
run @ zone.js:134
runOutsideAngular @ core.mjs:25486
(anonymous) @ core.mjs:26246
invoke @ zone.js:372
onInvoke @ core.mjs:25587
invoke @ zone.js:371
run @ zone.js:134
(anonymous) @ zone.js:1276
invokeTask @ zone.js:406
onInvokeTask @ core.mjs:25574
invokeTask @ zone.js:405
runTask @ zone.js:178
L @ zone.js:582
Promise.then (async)
d @ zone.js:565
scheduleTask @ zone.js:396
scheduleTask @ zone.js:221
scheduleMicroTask @ zone.js:241
F @ zone.js:1266
then @ zone.js:1410
bootstrapModule @ core.mjs:26167
9112 @ main.ts:10
_ @ bootstrap:19
(anonymous) @ main.ts:11
_.O @ chunk loaded:23
(anonymous) @ main.ts:11
s @ jsonp chunk loading:71
(anonymous) @ main.e6492dc8e37bae4b.js:1
main.ts:11 Error: Context is missing some of the required parameters
    at m.initRoot (context.ts:81:13)
    at new _ (app.component.ts:37:18)
    at _.ɵfac [as factory] (app.component.ts:65:23)
    at ss (core.mjs:3565:44)
    at Y_ (core.mjs:10179:23)
    at dh (core.mjs:12280:23)
    at Oc.create (core.mjs:21633:25)
    at e.bootstrap (core.mjs:26489:42)
    at core.mjs:26172:64
    at Array.forEach (<anonymous>)
(anonymous) @ main.ts:11
invoke @ zone.js:372
run @ zone.js:134
(anonymous) @ zone.js:1276
invokeTask @ zone.js:406
runTask @ zone.js:178
L @ zone.js:582
Promise.then (async)
d @ zone.js:565
scheduleTask @ zone.js:396
scheduleTask @ zone.js:221
scheduleMicroTask @ zone.js:241
F @ zone.js:1266
then @ zone.js:1410
bootstrapModule @ core.mjs:26167
9112 @ main.ts:10
_ @ bootstrap:19
(anonymous) @ main.ts:11
_.O @ chunk loaded:23
(anonymous) @ main.ts:11
s @ jsonp chunk loading:71
(anonymous) @ main.e6492dc8e37bae4b.js:1
core.mjs:6485 ERROR Error: Uncaught (in promise): Error: Cannot match any routes. URL Segment: 'edit/new:administration'
Error: Cannot match any routes. URL Segment: 'edit/new:administration'
    at er.noMatchError (router.mjs:2939:16)
    at router.mjs:2921:28
    at catchError.js:10:39
    at E._error (OperatorSubscriber.js:19:21)
    at E.error (Subscriber.js:40:18)
    at E._error (Subscriber.js:64:30)
    at E.error (Subscriber.js:40:18)
    at E._error (Subscriber.js:64:30)
    at E.error (Subscriber.js:40:18)
    at E._error (Subscriber.js:64:30)
    at G (zone.js:1213:31)
    at G (zone.js:1167:17)
    at zone.js:1279:17
    at T.invokeTask (zone.js:406:31)
    at Object.onInvokeTask (core.mjs:25574:33)
    at T.invokeTask (zone.js:405:60)
    at I.runTask (zone.js:178:47)
    at L (zone.js:582:35)
```

**Your environment** 
* **2sxc version(s):** 14.12.2
* **DNN:** [ 9.11 ]

**Comment by eXistenZe** [comment]
After searching the documentation I found this reference:

`GetService<ToSic.Sxc.Services.IPageService>().Activate("2sxc.JsCms");`

Which fixes this issue. 
Documentation needs some updates, with special attention to breaking changes.

**Comment by iJungleboy** [comment]
Thanks, we'll look into this.

**Comment by iJungleboy** [comment]
BTW: You can shorten this in v14 (assuming your base class is Hybrid.Razor14 to `Kit.Page.Activate("...")`

**Comment by iJungleboy** [comment]
Turns out it was a bug - your `Edit.Enable` should have worked as you expected. Fixed in 15.04

**Comment by iJungleboy** [comment]
thanks for reporting this 🥳
